package.json

```diff
@@ -21,31 +21,38 @@
     }
   ],
   "engines": {
-    "node": "12.0.x",
+    "node": "14.17.x",
```

Comment:
This is a bump from Minimum Discord required version, to the Long Term Support version of Node.